pkg/contracts/script/UpgradeCVMultichain.s.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "./BaseMultiChain.s.sol";
+import {RegistryFactoryV0_1} from "../src/RegistryFactory/RegistryFactoryV0_1.sol";
+
+contract UpgradeCVMultichain is BaseMultiChain {
+    using stdJson for string;
+
+    function runCurrentNetwork(string memory networkJson) public override {
+        // REGISTRY FACTORY UPGRADES
+        // address registryFactoryProxy = networkJson.readAddress(getKeyNetwork(".ENVS.PROXIES.REGISTRY_FACTORY"));
+        // Upgrades.upgradeProxy(
+        //     address(registryFactoryProxy),
+        //     "RegistryFactoryV0_1.sol",
+        //     abi.encodeWithSelector(RegistryFactoryV0_1.initializeV2.selector)
+        // );
+
+        // REGISTRY COMMUNITIES UPGRADES
+        // address[] memory registryCommunityProxies =
+        //     networkJson.readAddressArray(getKeyNetwork(".ENVS.PROXIES.REGISTRY_COMMUNITIES"));
+        // for (uint256 i = 0; i < registryCommunityProxies.length; i++) {
+        //     Upgrades.upgradeProxy(
+        //         address(registryCommunityProxies[i]),
+        //         "RegistryCommunityV0_1.sol",
+        //         abi.encodeWithSelector(RegistryCommunityV0_1.initializeV2.selector)
+        //     );
+        // }
+
+        // CV STRATEGIES UPGRADES
+        // address[] memory cvStrategyProxies = networkJson.readAddressArray(getKeyNetwork(".ENVS.PROXIES.CV_STRATEGIES"));
+        // for (uint256 i = 0; i < cvStrategyProxies.length; i++) {
+        //     Upgrades.upgradeProxy(
+        //         address(cvStrategyProxies[i]),
+        //         "CVStrategyV0_1.sol",
+        //         abi.encodeWithSelector(CVStrategyV0_1.initializeV2.selector, SENDER)
+        //     );
+        // }
```

Comment:
we are leaving this commented code to have it as an example on how to do it right

Reply:
I actually started with a V0_1 contract and did the upgrade script after but because we had private fields and functions, we needed to update the V0_0 directly so no more need for a next version yet. I was thinking keeping it for the next time yeah.